src/providers/UserActivityProvider/docs/UserActivityProvider.stories.mdx

```diff
@@ -0,0 +1,44 @@
+<Meta title="SDK Providers/UserActivityProvider" />
+
+# UserActivityProvider
+
+The `UserActivityProvider` provides the state of user activity.
+
+### State
+
+```javascript
+{
+    // Whether or not the the user is active
+    isUserActive: boolean | null;
+}
+```
+
+## Importing
+
+```javascript
+import { UserActivityProvider } from 'amazon-chime-sdk-component-library-react';
+```
+
+You can access the state by using the [useUserActivityState](/?path=/docs/sdk-hooks-useuseractivitystate--page) hook.
```

Comment:
not valid in the deployed documetnation because Im creating this documentation page as part of this PR, so it works when I start storybook locally, but its not in public docs yet

Reply:
Got it thanks for explaining